pkg/parser/parser.go

```diff
@@ -226,6 +228,9 @@ func (p *parser) Parse(tx string) any {
 		logger.Trace().Msg("no request word found")
 		return &brevity.UnableToUnderstandRequest{Callsign: pilotCallsign}
 	}
+	if !foundPilotCallsign && foundRequestWord && requestWord == picture {
+		return &brevity.PictureRequest{Callsign: nullCallsign}
```

Comment:
Just use empty string rather than "NULL". I'm pretty sure this is only here for logging purpose anyway.

Reply:
removed